web-ui/view/build.ml

```diff
@@ -101,3 +107,73 @@ let tabulate_steps step_rows =
     div
       ~a:[ a_class [ "container-fluid mt-8 flex flex-col space-y-6" ] ]
       [ div ~a:[ a_class [ "table-container" ] ] step_rows ])
+
+let repo_row ~repo_title ~short_hash ~last_updated ~status ~repo_uri =
```

Comment:
I would suggest that this should be in a `repo.ml` module. I think that the `build` module should only contain view elements and helpers relevant to the Build page.